README.md

```diff
@@ -211,6 +211,7 @@ Name | Website | Region
 [Fauna](/company-profiles/fauna.md) | https://www.fauna.com/ | Worldwide
 [Featurist](/company-profiles/featurist.md) | https://www.featurist.co.uk/ | Worldwide
 [Fetlife](/company-profiles/fetlife.md) | https://fetlife.com/ | Worldwide
+[FFW Agency](/company-profiles/ffwagency.md) | https://ffwagency.com/ | Worldwide
```

Comment:
Please change the file name here also to ffw-agency.md

Reply:
@dougaitken Oh, yes, sorry. I updated the README.md file. Thank you.